### ISSUE TITLE
(PC) feature flag(s) for MV to forbid/permit subdirs in data/content (blocked by MV #108)

### ISSUE THREAD

**Issue by ndushay** [sul-dlss/sdr-preservation-core#80]
This is for PC and depends on sul-dlss/moab-versioning#108.

We need a way to allow either strict or permissive Moab validation for:
- M2C seeding
- M2C existence check (when need to validate due to unexpected version found)
- C2M existence check (when need to validate due to unexpected version found)
- ...?

Possibilities:
- individual boolean settings in Settings.yml
- single default setting in Settings.yml, with individual overrides
- other??

This work is not done until:
1.  MV #108 is done and in a released version of the gem
2.  This code is done and merged to master.
3.  shared_configs for stage, prod are both set appropriately.  (could become a separate ticket)

**Comment by ndushay** [comment]
wrong repo!
